core/client/fs/src/main/java/alluxio/client/file/cache/PagedBlockReport.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache;
+
+public interface PagedBlockReport {
+  /**
+   * Report all block locations in a pageStoreDirs to master using BlockHeartbeat.
+   */
+  void reportBlocks();
+}
```

Comment:
I think a more general name like `CacheRestorationCallback` would be good. And the method name can be `onCacheRestorationSuccess`.